classification/management/commands/remove_invisible_characters.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from django.core.management.base import BaseCommand
+import re
+from classification.models import Classification
+from library.guardian_utils import admin_bot
+
+
+def has_invisible_characters(text):
+    pattern = re.compile(r'[^\u0000-\u007F]')
+    return re.search(pattern, text)
+
+
+def ensure_string(data):
+    if isinstance(data, (dict, list)):
+        return json.dumps(data)
+    elif isinstance(data, str):
+        return data
+    else:
+        return str(data)
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        classifications = Classification.objects.all()
+        user = admin_bot()
+
+        for classification in classifications:
+            evidence = classification.evidence
+
+            for key, value in evidence.items():
```

Comment:
Not a big deal, but you might as well just json.dumps all of evidence and check the result string, rather than iterating one level deep, then converting the next level to string

Reply:
Bharath explained that this was for debugging purposes so we can print out which fields will be affected - is worth keeping in